tests/keras_contrib/layers/test_crf.py

```diff
@@ -4,8 +4,12 @@
 
 from keras.utils.test_utils import keras_test
 from keras.layers import Embedding
+from keras.models import Sequential, model_from_json, load_model
+from keras.models import load_model
+from keras_contrib.losses import crf_loss
+from keras_contrib.metrics import crf_accuracy, crf_marginal_accuracy, crf_viterbi_accuracy
 from keras_contrib.layers import CRF
-from keras.models import Sequential, model_from_json
+from keras_contrib import *
```

Comment:
Please do not use wildcard imports. Import only what is needed.

Reply:
This is suggested by the `keras-contrib`  github home page, to handle `load_model` issue. But it does not seem to work.

Reply:
I cannot manage to successfully using `keras.models.load_model` to load model from `keras-contrib`.  If you have a working example,  would you mind sharing with me?

Reply:
Hmm, I haven't used it much myself. Simply passing the custom objects should be fine, but I've never tried it.

Reply:
@linxihui There is a workaround for the CRF layer here on #125.

Reply:
This is the only issue remaining before I merge the PR. Is there no workaround than to do a wildcard import?

Reply:
The solution/workaround I posted here (#129 ) seems to work. I hope it could be useful to @linxihui